_security/access-control/permissions.md

```diff
@@ -65,6 +65,18 @@ Rather than individual permissions, you can often achieve your desired security
 {: .tip }
 
 
+## System permission
+
+The system permission `system:admin/<system_index_name>` is unique among other permissions in that it extends some traditional admin-only accessibility to non-admin users. This permission gives normal users the ability to modify the system index specified in the permission name. For example, the permission `system:admin/.opendistro-alerting-config` gives a user permission to modify the system index that stores configurations for the Alerting plugin. 
+
```

Comment:
Apologies for the confusion, I stand corrected here. You were right in your review of raw materials. The way we setup system index permission is as follows:
1. In `index_permissions` section of a role:
     a. under `allowed_actions` section add `system:admin/system_index` 
     b. then under `index_patterns` section add the name of the index `.opendistro-alerting-config`
     
So the role would look something like this:
```
extensions_role:
  description: "ExtensionsRole"
  cluster_permissions:
    - '*'
  index_permissions:
    - index_patterns:
        - '.opendistro-alerting-config'
      allowed_actions:
        - '*'
        - 'system:admin/system_index'
```

Reply:
Note, that this doesn't affect the non system-indices, their authorization proceeds as normal.
For example, say there is an index named `.opensearch-index` and there is role that has index pattern  like `.opensearch*`:

1. If the index is marked as system index: User attached to this role will face authorization error
2. If the index is NOT marked as system index : User action will proceed as normal

This suggests that `*` pattern cannot be used with system-indices, they have to be individually listed.

Hope this clarifies any confusion.

Reply:
I think I have everything in this comment covered in recent changes.